### PR TITLE
Create user mutation

### LINKED ISSUE #19: Create User mutation
**Chore**
- [x] In mutations folder, make user file
- [x] Create User--fill in information
- [x] Make test
- [x] Update the GraphQL gist with a sample query and response

### PR DESCRIPTION
**Pull request by rer7891**
### Sidebar Checklist

- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project

### Issues Resolved
Resolves #19 

### Problem Addressed
need a way to create a user 
### Solution Implemented
created a create_user mutation and tested it. 
### Other Notes
will add this to our documentation

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -103,5 +103,5 @@
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
   # Rack-CORS allowed origin in this environment
-  #config.allowed_cors_origins = ["https://landslide-57f9a.firebaseapp.com"]
+  config.allowed_cors_origins = ["http://localhost:3000"]
```
Comment (tyladevon) [inline config/environments/production.rb:106]: Nice fix here  Becky.

File: app/graphql/mutations/users/create_user.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module Mutations
+  module Users
+    class CreateUser < ::Mutations::BaseMutation
+      argument :first_name, String, required: true
+      argument :last_name, String, required: true
+      argument :email, String, required: true
+      argument :google_token, String, required: true
```
Comment (tyladevon) [inline app/graphql/mutations/users/create_user.rb:7]: For phase two, we may add in user's est-time for self care which would be also input here, correct?

**Comment by wrenisit** [comment]
Thank you for getting this going! It looks great.

File: app/graphql/mutations/users/create_user.rb (hunk 2 above)
Comment (joel-oe-lacey) [reply app/graphql/mutations/users/create_user.rb:7]: We have an accessToken, an IDToken and a refreshToken, I'm not too familiar but something for us to dig into to see which ones we want to store. We might need to modify structure accordingly.

**Pull request merged**